src/include/daos_mgmt.h

```diff
@@ -77,6 +79,10 @@ typedef struct {
 int
 daos_pool_stop_svc(daos_handle_t poh, daos_event_t *ev);
 
+int
+daos_mgmt_list_pools(const char *group, daos_size_t *npools, daos_mgmt_pool_info_t *pools,
+		     daos_event_t *ev);
+
```

Comment:
there is no API description. so should add that.
I assume one can query the size with NULL pools buf, or provide an estimated buffer of certain size and the call return the actual size in npools?
this also needs some C tests added.

Reply:
> there is no API description. so should add that. 
Done.

> I assume one can query the size with NULL pools buf, or provide an estimated buffer of certain size and the call return the actual size in npools? 
Correct. This is following the pattern established by other pool client APIs.

> this also needs some C tests added.
Ack. Will add them.